cdap-watchdog/src/test/java/co/cask/cdap/logging/clean/FileMetadataCleanerTest.java

```diff
@@ -440,7 +298,52 @@ public void testFileMetadataWithCommonContextPrefix() throws Exception {
       }
     } finally {
       // cleanup meta
-      cleanupMetadata(transactional, datasetManager);
+      deleteAllMetaEntries(transactionRunner);
+    }
+  }
+
+  private boolean deleteAllMetaEntries(TransactionRunner transactionRunner) {
+    final List<DeletedEntry> deletedEntries = new ArrayList<>();
+    try {
+      TransactionRunners.run(transactionRunner, context -> {
+        Stopwatch stopwatch = new Stopwatch();
+        stopwatch.start();
+        StructuredTable table = context.getTable(StoreDefinition.LogFileMetaStore.LOG_FILE_META);
+        try (CloseableIterator<StructuredRow> iter = table.scan(Range.all(), Integer.MAX_VALUE)) {
+          while (iter.hasNext()) {
+            if (stopwatch.elapsedTime(TimeUnit.SECONDS) > CUTOFF_TIME_TRANSACTION) {
```

Comment:
Do we need a cutoff time here? How many files does this test generate?

Reply:
This is only used for cleaning existing entries for next test.